src/lib/core/datetime.c

```diff
@@ -15,8 +15,8 @@
 #include "tzcode/tzcode.h"
 
 /** floored modulo and divide */
-#define MOD(a, b) unlikely(a < 0) ? (b + (a % b)) : (a % b)
-#define DIV(a, b) unlikely(a < 0) ? ((a - b + 1) / b) : (a / b)
+#define MOD(a, b) (unlikely(a < 0) ? (b + (a % b)) : (a % b))
+#define DIV(a, b) (unlikely(a < 0) ? ((a - b + 1) / b) : (a / b))
```

Comment:
Those macros mark places where we intentionally need to proceed _floored modulo_ (similar as Lua does for us in 5.1+ `%` operator) instead of a default C/C++ modulo which would produce negative modulo values for negative dividend. 

Historically, there was wider usage of those macros in `datetime*-wip` branch (i.e. I needed it in SQL builtins implementation), but you'll later learn it when needing.